srcpkgs/scons/template

```diff
@@ -1,18 +1,30 @@
 # Template file for 'scons'
 pkgname=scons
-version=4.0.1
-revision=3
-wrksrc="SCons-${version}"
-build_style=python3-module
-hostmakedepends="python3 python3-setuptools"
+version=4.4.0
+revision=1
+#build_style=python3-module
+hostmakedepends="python3 python3-setuptools python3-psutil python3-wheel python3-pytest tar"
 depends="python3"
 short_desc="Software construction tool"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Wilson Birney <wpb@360scada.com>"
 license="MIT"
 homepage="https://www.scons.org/"
-distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
-checksum=722ed104b5c624ecdc89bd4e02b094d2b14d99d47b5d0501961e47f579a2007c
+distfiles="https://github.com/SCons/scons/archive/${version}.tar.gz>${pkgname}-${version}.tar.gz"
+checksum=b4fef9df167b16b94a91bcc3905c595a5768a433f2aee9abe7f4fffac201eeaa
 
+do_build() {
+	python3 scripts/scons.py --include-dir=
```

Comment:
What wrong with good old `python3 setup.py`???

Reply:
--include-dir is not needed, ill remove that in the next push. I copied that from archlinux's pkgbuild when it was having issues. 

Thanks!

Reply:
as for the python3 setup.py, that doesnt work with scons. they require you run scripts/scons.py *prior* to python3 setup.py

Reply:
I am actually incorrect in this, scripts/scons.py is for building from the repo not the release tarball, correcting now and pushing. My bad